src/utils/components/DiskModal/utils/form.ts

```diff
@@ -0,0 +1,93 @@
+import DataSourceModel from '@kubevirt-ui/kubevirt-api/console/models/DataSourceModel';
+import {
+  V1DataVolumeTemplateSpec,
+  V1VirtualMachine,
+  V1Volume,
+} from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { getName } from '@kubevirt-utils/resources/shared';
+import {
+  getBootDisk,
+  getDataVolumeTemplates,
+  getDisks,
+  getVolumes,
+} from '@kubevirt-utils/resources/vm';
+import { isInstanceTypeVM } from '@kubevirt-utils/resources/vm/utils/instanceTypes';
+import { generatePrettyName, isEmpty } from '@kubevirt-utils/utils/utils';
+import { isRunning } from '@virtualmachines/utils';
+
+import { DEFAULT_DISK_SIZE } from './constants';
+import { requiresDataVolume } from './helpers';
+import { InterfaceTypes, SourceTypes, V1DiskFormState } from './types';
+
+const getDefaultDataVolumeTemplate = (name: string): V1DataVolumeTemplateSpec => ({
+  metadata: { name },
+  spec: { source: {}, storage: { resources: { requests: { storage: DEFAULT_DISK_SIZE } } } },
```

Comment:
can use selector instead of this "pretty" deconstruct :)

Reply:
here we are trying to create a new data volume template. I'm not getting one from the state. Don't know what do you mean for selector